GTG/gtk/editor/__init__.py

```diff
@@ -31,8 +31,8 @@ class GnomeConfig:
 
     MARK_DONE = _("Mark as Done")
     MARK_UNDONE = _("Mark as not Done")
-    MARK_DISMISS = _("Dismiss")
-    MARK_UNDISMISS = _("Undismiss")
+    MARK_DISMISS = _("Dismiss Task")
+    MARK_UNDISMISS = _("Undismiss Task")
 
     MARK_DONE_TOOLTIP = _("Mark this task as done")
     MARK_UNDONE_TOOLTIP = _("Mark this task as to be done")
```

Comment:
@izidormatusov well, shouldn't all of them be placed to a .ui file?

I mean, I can create, as you suggest a greater variety of unused (property visible == False) widgets which will be inter-changed with those default ones and set all the tooltip properties automatically there and all of these can go away, right?